app/models/image-set-marker.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Object.extend({
+  imageSet: null,
+  lat: Ember.computed.alias('imageSet.latitude'),
```

Comment:
instead of relying on two-way computed properties to update `imageSet.latitude` and longitude, the ember 2.0 way to do this would be to use an action. The `lg-google-map` component would be given an action and the observers for marker lat and lng would just fire that action with an updated [lat,lng] tuple. Bonus points for using debounce and an config'd timeout to avoid overloading things.